js/components/controlBar.js

```diff
@@ -22,6 +22,10 @@ var ControlBar = React.createClass({
     this.isMobile = this.props.controller.state.isMobile;
     this.responsiveUIMultiple = this.getResponsiveUIMultiple(this.props.responsiveView);
     this.moreOptionsItems = null;
+    this.vr = false;
+    if (this.props.controller && this.props.controller.videoVrSource) {
+      this.vr = this.props.controller.videoVrSource;
```

Comment:
Is there a reason why we copy this to local state instead of using the prop directly? Any changes to this object will not be propagated to the component unless it is remounted.

Reply:
Thank you for the comment. We will change it in userstory 1424

Reply:
Thank you for the comment. We will change it in userstory 1424